Sources/Components/SelectionBoxes/SelectionboxItem.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public class SelectionboxItem: UIView {
+
+    // MARK: Internal properties
+
+    let imageView: AnimatedImageView = {
+        let view = AnimatedImageView(frame: .zero)
+        view.contentMode = .scaleAspectFit
+        view.animationRepeatCount = 1
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    let label: UILabel = {
```

Comment:
I think we should rename this as "titleLabel" or "headingLabel" just label by itself doesn't have much meaning.